src/styles/main.scss

```diff
@@ -1,3 +1,24 @@
 body {
   margin: 0;
 }
+$color-grey-light: #f3f3f3;
+$color-white: #fff;
+$color-accent-blue: #00acdc;
+$color-secondary: #616070;
+$color-main-accent: #060b35;
+$color-hover-accent: #34568b ;
+
+body,
+p,
+h2 {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+@import "components/product-card";
+@import "components/stars";
+@import "components/header";
+@import "components/product-grid";
```

Comment:
I think, you can create variables and rest files. Variables file for your variables, reset file for box-sizing and other styles for body, p, etc. In main file you can save only @import. If you want, you can do this.